ci/prod/build/frontend.Dockerfile

```
@@ -0,0 +1,22 @@
+FROM oven/bun:1.2-slim
+
+WORKDIR /app
+
+RUN apt-get update && apt-get install -y --no-install-recommends ca-certificates
```

Hadolint (2.12.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)